server/types.graphql

```diff
@@ -32,3 +32,19 @@ type User @model {
 #   # Every relation also required a back-relation (to determine 1:1, 1:n or n:m)
 #   author: User! @relation(name: "UserPosts")
 # }
+
+type Project @model {
+  # Required system field:
+  id: ID! @isUnique # read-only (managed by Graphcool)
+
+  # Optional system fields (remove if not needed):
+  createdAt: DateTime! # read-only (managed by Graphcool)
+  updatedAt: DateTime! # read-only (managed by Graphcool)
+
+  name: String!
+  description: String
+  repo: String #url
```

Comment:
My initial thought with these is that each of these options could potentially be plural (e.g. multiple associated repos), however, I personally think that case is very uncommon, and if we run into it we could either add additional fields to support that, or simply have projects describe those cases in the `description` field.

Reply:
Agreed. Owlet is the only one that comes to mind.